packit_service/worker/tasks.py

```diff
@@ -40,3 +49,43 @@
 @celery_app.task(name="task.steve_jobs.process_message")
 def process_message(event: dict, topic: str = None) -> Optional[dict]:
     return SteveJobs().process_message(event=event, topic=topic)
+
+
+@celery_app.task(
+    bind=True,
+    name="task.babysit_copr_build",
+    # 120 minutes
+    default_retry_delay=120 * 60,
```

Comment:
This is the time when we start this baby-sit task for the first time? (Any then retry each minute?) (i.e. 120, 1, 1, 1, ...)


I would try some exponential sequence as Dominika suggested. (e.g. 1, 2, 4, 8, 16, ... ) or, a constant + exponential sequence (e.g. 10, 1, 2, 4, 8, ... )

[examples are in minutes and mean time to the next check]

Reply:
I'm not too sold on the exponential backoff